### PR TITLE
cleanup deprecated deck GCS URLs

### PR DESCRIPTION
**Pull request by BenTheElder**
This is a not-quite-exhaustive cleanup of /view/gcs -> /view/gs (as we've deprecated /view/gcs).

### CODE REVIEW

**Comment by fejta** [comment]
Might be worth breaking up this PR so it has fewer conflicts?

**Review by fejta — APPROVED** [review]

**Comment by BenTheElder** [comment]
rebased

File: prow/spyglass/README.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ The following lenses are available:
 deck:
   spyglass:
     size_limit: 100000000  # 100 MB
-    gcs_browser_prefix: https://gcsweb.k8s.io/gcs/
+    gcs_browser_prefix: https://gcsweb.k8s.io/gs/
```
Comment (fejta) [inline prow/spyglass/README.md:78]: gcsweb also now accepts the gs prefix??

**Review by fejta — APPROVED** [review]
/hold

File: prow/spyglass/README.md (hunk 1 above)
Comment (BenTheElder) [reply prow/spyglass/README.md:78]: bah. no.

Comment (BenTheElder) [reply prow/spyglass/README.md]: fixed, and also the line above, but not the line before that (which is plank)

**Review by fejta — APPROVED** [review]

**Pull request merged**